<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBoxLabelPrint</class>
 <widget class="QDialog" name="DialogBoxLabelPrint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>198</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>彩盒标打印</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>请扫描设备上的条码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDevSN">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="placeholderText">
    <string>请扫描机身条形码（焦点在此框）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>开始打印</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAutoPrint">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>扫描完成后自动打印</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
